ssv/spectest/run_test.go

```diff
@@ -39,6 +40,10 @@ func TestAll(t *testing.T) {
 	wait.Wait()
 }
 
+// TestOne will run a single test.
+func TestOne(t *testing.T) {
+	proposer.ProposeBlindedBlockDecidedRegular().Run(t)
+}
```

Comment:
By the name and the file location, I was under the impression this function will implement generally running a single test, any single test. though it seems like it runs a specific single test which I'm not sure why and why it is here. is it a mistake? can you explain @GalRogozinski ?

Reply:
this shouldn't be here
I committed it accidentally

Thanks